applications/StructuralMechanicsApplication/structural_mechanics_application_variables.h

```diff
@@ -175,6 +175,8 @@ typedef std::vector<MpcDataPointerType>
 
     // For MPC implementations
     KRATOS_DEFINE_VARIABLE(MpcDataSharedPointerVectorType, MPC_DATA_CONTAINER)
+
+	KRATOS_DEFINE_APPLICATION_VARIABLE(STRUCTURAL_MECHANICS_APPLICATION, double, INELASTIC_FLAG)
```

Comment:
Flag double?

Reply:
I checked the usage of this and I would recommend to use bool

Especially if you later want to check if it is true or false this is way easier with bool than with double